astropy/time/tests/test_basic.py

```diff
@@ -952,6 +952,25 @@ def test_copy(self):
         assert t.location.x == t_loc_x  # prove that it changed
 
 
+class TestStardate:
```

Comment:
A CLASS?!?

Reply:
How would you have done it?

Reply:
exactly the same but without the class definition.

Reply:
@Cadair doesn't believe in namespaces

Reply:
Lol, well a bunch of the other tests were already classes so I did it like that too :man_shrugging:

Reply:
I think this is fine as a class.  Mainly because it makes me think of tests my class at Starfleet Academy had to work together to pass... 🌟